### PR TITLE
DOC: Centralize documentation.md

### PR DESCRIPTION
**Pull request by ke-zhang-rd**
This is motivated by https://github.com/bluesky/suitcase-core/issues/76

### CODE REVIEW

**Comment by ke-zhang-rd** [comment]
This PR include a Quick Start example try to avoid duplicate central suitcase docs.

File: README.md [hunk 1]
```diff
@@ -7,3 +7,27 @@ This is a suitcase subpackage for reading a particular file format.
...
+## Quick Start
+
+`suitcase-tiff` supports stack and series.
+
+Using stack:
+
+```
+import suitcase.tiff_stack
+docs = db[-1].documents(fill=True)
+suitcase.tiff_stack.export(, 'my_exported_files/', file_prefix='PREFIX-')
```
Comment (danielballan) [inline README.md:20]: Missing the first argument, which should be `docs`

File: README.md [hunk 2]
```diff
@@ -7,3 +7,27 @@ This is a suitcase subpackage for reading a particular file format.
...
+```
+
+Using series:
+
+```
+import suitcase.tiff_series
+docs = db[-1].documents(fill=True)
+suitcase.tiff_series.export(, 'my_exported_files/', file_prefix='PREFIX-')
```
Comment (danielballan) [inline README.md:28]: Same here

File: README.md (hunk 1 above)
Comment (ke-zhang-rd) [reply README.md:20]: thanks

**Pull request merged**